sp-sim/src/config.rs

```diff
@@ -17,15 +17,64 @@ use std::path::Path;
 use std::path::PathBuf;
 use thiserror::Error;
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub enum NetworkConfig {
```

Comment:
Tiny nit - could we drop a `#[serde(rename_all = "snake_case")]` on this so the TOML keys are lowercase like the others in the config files?

Reply:
done in e76c992e15e21adcff7be849e79eb532202136d6